src/redux/services/resourceRefs.ts

```diff
@@ -68,6 +63,14 @@ const addRefNodeAtPath = (refNode: RefNode, path: string, refNodesByPath: Record
   }
 };
 
+export function refMapperMatchesKind(refMapper: RefMapper, kind: string) {
+  if (refMapper.target.kind.startsWith('$')) {
+    return kind.match(refMapper.target.kind.substring(1)) !== null;
```

Comment:
I'm wondering if it would be safer/faster to split by `|` and check each value manually instead of regex, but we could leave it like this to start with

Reply:
yes let's leave it for now - we discussed adding the apiVersion to this also eventually - which we could cover with regex expressions also